spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraTableRowReaderProvider.scala

```diff
@@ -29,13 +29,13 @@ trait CassandraTableRowReaderProvider[R] {
 
   protected def readConf: ReadConf
 
-  protected def fetchSize: Int = readConf.fetchSize
+  protected def fetchSize: Int = readConf.fetchSizeInRows
```

Comment:
That's a good idea but even a better idea would be to encode the unit in the type. And even "more better" idea is to not code it by ourselves, but use something already invented for this purpose e.g. squants http://www.squants.com/. However, we should use it in all places and not just this one, so let's do it in a separate ticket.